package.txt

```diff
@@ -1,7 +1,7 @@
 NAME=bison
-VERSION=3.0.5
-REVISION=2
-SRC=http://mirrors.kernel.org/gnu/bison/bison-3.0.5.tar.xz
-SRCMD5=3e54f20988ecd1b62044e25481e5f06b
+VERSION=3.1
+REVISION=1
```

Comment:
@shedbuilt-jb As this is the third revision to the bison packaging for blank, the revision should be 3.